common/k8stest/util_deploy.go

```diff
@@ -466,3 +466,15 @@ func RestoreDeploymentReplicas(deploymentName string, namespace string, timeout_
 	}
 	return err
 }
+
+func VerifyDeploymentReadyReplicaCount(name, namespace string, expectedReplicas int) (bool, error) {
+	deployment, err := GetDeployment(name, namespace)
+	if err != nil {
+		return false, err
+	}
+	logf.Log.Info("Creating Snapshot", "Ready Replicas", deployment.Status.ReadyReplicas, "Replicas Expected", expectedReplicas)
```

Comment:
Log message seems incorrect , there is no snapshot here

Reply:
Fixed